presto-docs/src/main/sphinx/functions/khyperloglog.rst

```diff
@@ -0,0 +1,75 @@
+======================
+KHyperLogLog Functions
+======================
+
+Presto implements the `KHyperLogLog <https://research.google/pubs/pub47664/>`_
+algorithm and data structure. KHyperLogLog data structure can be created
+through :func:`khyperloglog_agg` function.
+
+
+Data Structures
+---------------
+
+KHyperLogLog is a data sketch that compactly represents the association of two
+columns. It is implemented in Presto as a two-level data structure composed of
+a MinHash structure whose entries map to HyperLogLog sketches.
```

Comment:
Double backquotes:
```
``MinHash``
``HyperLogLog``
```

Reply:
All these references to MinHash and HyperLogLog are referring to data structures and not some type in Presto.  I'm indifferent either way, but I was under the impression we'd use code formatting for Presto types that one might use in a SQL statement.  Or in other words, what's the rule for when we use monoformat?

Reply:
makes sense